tsconfig.json

```diff
@@ -10,6 +10,11 @@
     "esModuleInterop": true,                        /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
     /* Advanced Options */
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
+<<<<<<< HEAD
```

Comment:
you missed resolving the conflicts i guess @iifawzi

Reply:
Resolved.